tests/unit/tabs/tabs_options.js

```diff
@@ -141,6 +142,12 @@ test( "{ collapsible: true }", function() {
 	element.find( ".ui-state-active .ui-tabs-anchor" ).click();
 	equal( element.tabs( "option", "active" ), false );
 	state( element, 0, 0, 0 );
+
+	element.tabs( "option", "collapsible", false );
+	ok( !element.hasClass( "ui-tabs-collapsible" ), "main element is not .ui-tabs-collapsible" );
```

Comment:
Okay, in that case, let's add a second assertion. I don't mind if the implementation duplicates code, as long as using the assertion is as simple as `assert.hasClasses` is now.